### PR TITLE
[UX specs] Create Backend page

### PR DESCRIPTION
**Pull request by deernano**
**What this PR does / why we need it**:

>  UX spec for  Backend > Create_Backend page

**Which issue(s) this PR fixes** 

> Jira Issue: https://issues.redhat.com/browse/APPDUX-343
Marvel link : https://marvelapp.com/prototype/12844cg4/screen/71559193

**Verification steps** 

**Special notes for your reviewer**:

### CODE REVIEW

File: portafly/doc/ux_specs/backend/create_backend.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      * Name already exists on _onblur_
+  * System name [required] > text input w/ placeholder and helper text
+    * Inline help:
+      * Label include a `pf-icon-help` icon
+      * Icon toggles a popover ([PF specs](https://www.patternfly.org/v4/documentation/react/components/popover))
+      * Popover includes content about
+        * the need for the system name to be unique
+        * an alert to users about the system name not being editable once the product is created
+    * Validation errors:
+      * Not valid name on form submit
```
Comment (dfennessy) [inline portafly/doc/ux_specs/backend/create_backend.md:27]:
```suggestion
      * Not a valid name on form submit
```

File: portafly/doc/ux_specs/backend/create_backend.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      * Name already exists on form submit
+      * Not valid name on _onblur_
```
Comment (dfennessy) [inline portafly/doc/ux_specs/backend/create_backend.md:29]:
```suggestion
      * Not a valid name on _onblur_
```

File: portafly/doc/ux_specs/backend/create_backend.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Create Backend
+
+[Back to index](../index.md)
+
+* [Mockups](https://marvelapp.com/prototype/12844cg4)
+* [Mockups JIRA ticket](https://issues.redhat.com/browse/APPDUX-343)
+* [Parent JIRA ticket](https://issues.redhat.com/browse/THREESCALE-5575)
+
+##### Title
+* Create a new backend
```
Comment (cntlsn) [inline portafly/doc/ux_specs/backend/create_backend.md:10]: @deernano We don't need to specify the title in the UX specs. The title will be specified in the UI strings.

File: portafly/doc/ux_specs/backend/create_backend.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+##### Form
+* Input fields
+  * Name > text input [required]
+    * Validation errors:
+      * Required field on form submit (if left blank)
+      * Name already exists on form submit
+      * Name already exists on _onblur_
```
Comment (cntlsn) [inline portafly/doc/ux_specs/backend/create_backend.md:18]:
```suggestion
  * Name [required] > text input
    * Validation errors ([specs](../../global_components/alerts.md)):
      * Required field on form submit (if left blank)
    * Validation alerts ([specs](../../global_components/alerts.md)):
      <!-- * Name is already in use on form submit -->
      * Name is already in use recommendation on _onblur_
```

File: portafly/doc/ux_specs/backend/create_backend.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        * the need for the system name to be unique
+        * an alert to users about the system name not being editable once the product is created
+    * Validation errors:
+      * Not valid name on form submit
+      * Name already exists on form submit
+      * Not valid name on _onblur_
+      * Name already exists on _onblur_
+  * Description > text area
+    * NO validation errors
+  * API private base URL > text input w/ placeholder and helper text
```
Comment (cntlsn) [inline portafly/doc/ux_specs/backend/create_backend.md:33]:
```suggestion
  * API private base URL [required] > text input w/ placeholder and helper text
```

File: portafly/doc/ux_specs/backend/create_backend.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+        * the need for the system name to be unique
+        * an alert to users about the system name not being editable once the product is created
+    * Validation errors:
+      * Not a valid name on form submit
+      * Name already exists on form submit
+      * Not a valid name on _onblur_
+      * Name already exists on _onblur_
+  * Description > text area
+    * NO validation errors
+  * API private base URL [required] > text input w/ placeholder and helper text
```
Comment (dfennessy) [inline portafly/doc/ux_specs/backend/create_backend.md:31]:
```suggestion
  * API private base URL [required] > text input with placeholder and helper text
```

File: portafly/doc/ux_specs/backend/create_backend.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+##### Form
+* Input fields
+  * Name [required] > text input
+    * Validation errors ([specs](../../global_components/alerts.md)):
+      * Required field on form submit (if left blank)
+    * Validation alerts ([specs](../../global_components/alerts.md)):
+       <!-- * Name is already in use on form submit -->
+       * Name is already in use recommendation on _onblur_
+  * System name [required] > text input w/ placeholder and helper text
```
Comment (dfennessy) [inline portafly/doc/ux_specs/backend/create_backend.md:17]:
```suggestion
  * System name [required] > text input with placeholder and helper text
```

**Review by cntlsn — APPROVED** [review]

**Pull request merged**